extensions/amp-selector/1.0/example/index.html

```diff
@@ -1,36 +1,63 @@
 <!DOCTYPE html>
 <html>
   <head>
+    <title>Tabs with bento-selector</title>
     <script
       type="module"
       async
       src="https://cdn.ampproject.org/bento.mjs"
     ></script>
-    <script nomodule src="https://cdn.ampproject.org/bento.js"></script>
-    <script
-      type="module"
-      async
-      src="https://cdn.ampproject.org/v0/bento-selector-1.0.mjs"
-    ></script>
-    <script
-      nomodule
-      async
-      src="https://cdn.ampproject.org/v0/bento-selector-1.0.js"
-    ></script>
-    <link
-      rel="stylesheet"
-      type="text/css"
-      href="https://cdn.ampproject.org/v0/bento-selector-1.0.css"
-    />
+    <script src="https://cdn.ampproject.org/bento.js"></script>
+    <script async src="https://cdn.ampproject.org/v0/bento-selector-1.0.js" ></script>
+    <link rel="stylesheet" type="text/css" href="https://cdn.ampproject.org/v0/bento-selector-1.0.css" />
+<style>
+    :root {
+      --space-1: .5rem;  /* 8px */
+      --space-4: 2rem;   /* 32px */
+    }
+    body {
+      font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
```

Comment:
Is there a reason for this complex of a font-family? Seems unnecessary for the example

Reply:
That's just the system fonts. The alternative would be to load an extra font file.